### PR TITLE
el_CY Locale

### PR DESCRIPTION
**Pull request by softius**
That is Greek (Cyprus).

I've included localised Providers for Person, Company, Address, Internet, Payment and PhoneNumber.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -832,6 +832,18 @@ echo $faker->cvr; // "32458723"
 echo $faker->p; // "5398237590"
 ```
 
+### `Faker\Provider\el_CY\Payment`
+
+```php
+<?php
+
+// Generates a random bank name
+echo $faker->bank; // "Τράπεζα Κύπρου"
```
Comment (fzaninotto) [inline readme.md]: these two are generic, no need to include them here.

File: src/Faker/Provider/el_CY/Person.php [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+namespace Faker\Provider\el_CY;
+
+class Person extends \Faker\Provider\Person
+{
+    /**
+     * @var array
+     * @link http://www.mof.gov.cy/mof/cystat/statistics.nsf/populationcondition_22main_gr/populationcondition_22main_gr?OpenForm&sub=2&sel=2
+     */
+    protected static $firstNameMale = array(
+        'Άγγελος',
```
Comment (fzaninotto) [inline src/Faker/Provider/el_CY/Person.php]: There are not enough names for a good entropy

File: readme.md (hunk 1 above)
Comment (softius) [inline readme.md]: OK I will remove them

**Comment by fzaninotto** [comment]
Thanks!

**Pull request merged**